content/applications/hr/payroll/work_entries.rst

```diff
@@ -80,95 +93,186 @@ there are any errors on the work entry, such as required fields being left blank
-the time-off will be entered in the :guilabel:`Time Off` field. Clicking the :guilabel:`External
-Link` button next to the :guilabel:`Time Off` entry will bring up the duplicate time-off request.
+The time-off conflict can be resolved either on the work entry pop-up, or on a detailed time-off
+request pop-up. To resolve the time-off conflict on this work entry pop-up, click the
+:guilabel:`Approve Time Off` button to approve the time off request and resolve the work entry
+conflict.
 
-The details for the time-off request appear in the pop-up. The request can be modified if needed.
-Click either the :guilabel:`Validate` or :guilabel:`Refuse` button to approve or deny the request,
-then click the :guilabel:`Save` button to save the changes.
+The :guilabel:`Approve Time Off` and :guilabel:`Refuse Time Off` buttons disappear. Click the
+:guilabel:`Save & Close` button to close the pop-up. The conflict disappears from the work entry
+dashboard since the conflict is resolved.
 
-.. image:: work_entries/validate.png
+.. image:: work_entries/approve-work-entry.png
    :align: center
-   :alt: Edit and/or validate a duplicate time-off request.
+   :alt: The time-off conflict in the work entry detail pop-up.
 
-Once the duplicate time-off request has been approved and saved, the screen goes back to the
-conflict. Click :guilabel:`Refuse Time Off` or :guilabel:`Approve Time Off` via the buttons in the
-top right to either approve or deny the request. Repeat for all conflicts until there are no
-conflicts to resolve.
+Resolve on time off request
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-After conflicts have been resolved, work entries must be regenerated for each employee by clicking
-the :guilabel:`Regenerate Work Entries` button, and entering the corresponding information for each
-employee.
+To resolve the time-off conflict on the detailed time-off request pop-up, click the
+:guilabel:`Internal Link` button at the end of the :guilabel:`Time Off` entry line and the time-off
+request details appear in a pop-up window. The request can be modified if needed.
 
-.. image:: work_entries/regenerate-employee.png
+Click the :guilabel:`Approve` button to approve the request, then click the :guilabel:`Save & Close`
+button to save the changes and go back to the work entry conflict pop-up.
+
+.. image:: work_entries/time-off-details.png
    :align: center
-   :alt: Regenerate Work Entries button on the Work Entries Regeneration form.
+   :alt: The detailed time-off request form.
+
+Now, the :guilabel:`Approve Time Off` button is hidden, only the :guilabel:`Refuse Time Off` button
+is visible. If the approval was a mistake, the request can be refused here by clicking the
+:guilabel:`Refuse Time Off` button.
+
+Since the time-off was approved in the time-off window, click the :guilabel:`X` in the top right
+corner to close the window. The conflict disappears from the work entry dashboard since the conflict
+is resolved.
```

Comment:
```suggestion
Since the time off was approved in the time off window, click the :guilabel:`X` in the top right
corner to close the window. The conflict disappears from the work entry dashboard since it has been resolved.
```